src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -384,4 +384,18 @@ public void setCreated_NonProduction(Date createdAt) {
     public void setUpdatedAt_NonProduction(Date updatedAt) {
         this.updatedAt = updatedAt;
     }
+
+    /**
+     * Checks whether the edit form of student has changed the section/team values.
+     * @param originalStudentAttribute
+     * @return true if either section or team has changed from previous values. 
+     */
+    public boolean hasSectionOrTeamChanged(StudentAttributes originalStudentAttribute) {
+       if ((this.section!=null && !this.section.equals(originalStudentAttribute.section)) || 
```

Comment:
Thanks @JosephineKwa for the review. Nice catch. 
In this case, I don't see any other solution than refactoring the code and separating section and team validation calls to separate methods. In fact method name '`validateSections`' indicates it validates only "sections", where it actually also validates teams. I moved them in to two separate methods. Please have a re-look.